content/integrations/desk.md

```diff
@@ -3,6 +3,7 @@ title: Datadog-Desk Integration
 integration_title: Desk
 kind: integration
 git_integration_title: desk
+description: "See and discuss new, open, pending, and resolved cases in your event stream."
```

Comment:
Should "Desk.com" be mentioned in the description? Isn't this used by Google for indexing?

Reply:
will do it on enhancement